packages/widgets/src/commandpalette.ts

```diff
@@ -287,62 +287,64 @@ export class CommandPalette extends Widget {
    * A message handler invoked on an `'update-request'` message.
    */
   protected onUpdateRequest(msg: Message): void {
-    // Fetch the current query text and content node.
-    let query = this.inputNode.value;
-    let contentNode = this.contentNode;
-
-    // Ensure the search results are generated.
-    let results = this._results;
-    if (!results) {
-      // Generate and store the new search results.
-      results = this._results = Private.search(this._items, query);
-
-      // Reset the active index.
-      this._activeIndex = query
-        ? ArrayExt.findFirstIndex(results, Private.canActivate)
-        : -1;
-    }
+    if (!this.isHidden) {
```

Comment:
Could this condition be reversed to limit the number of lines changed (/indented), like

```
if (this.isHidden) {
   return;
}
```

?

Reply:
Sure! No idea why I didn't think of this :man_facepalming:

Reply:
Yes! No idea why I didn't think of that at first 🤦🏽‍♂️

Reply:
The indentations have now been reverted